src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        for (int i = figures.length / 2; i < figures.length; i++) {

```

Comment:
Consider using a single loop to fill the `figures` array with random figures for the first half and default figures for the second half. This would make the code more concise and maintainable.